Calypso-SystemPlugins-SUnit-Browser.package/ClyBrowseCorrespondingTestCaseCommand.class/instance/execute.st

```diff
@@ -0,0 +1,3 @@
+execution
+execute
+	browser spawnFullBrowser selectClass: classes first correspondingTestCase
```

Comment:
The class-side test checks `lastSelectedClass` which is actually the `first` item in the collection. But the command inherits from SyClassCommand so I am using the classes collection from the parent.